.github/workflows/build.yml

```diff
@@ -18,5 +18,4 @@ jobs:
         uses: ./.github/actions/build-sign-publish-chainlink
         with:
           publish: false
-          image-name: test-chainlink
```

Comment:
Should this have been removed? Should we use a different name for image-names that arent pushed?

Reply:
I couldn't find any instances where the image name seems to be referenced in this case, so I was thinking that `chainlink/chainlink` works from default

Not sure about adding it back as `ecr-image-name: test-chainlink` since it's not being pushed up to ECR